cfme/tests/cloud/test_cloud_object_tag_visibility.py

```diff
@@ -42,27 +42,30 @@ def testing_vis_object(request, a_provider, appliance):
     """
     collection_name, param_class = request.param
     if collection_name == 'stacks':
-        return param_class(name=a_provider.data['provisioning']['stacks'][0], provider=a_provider)
+        return param_class(appliance).instantiate(
+            name=a_provider.data['provisioning']['stacks'][0], provider=a_provider)
```

Comment:
**Optional**
Since `a_provider.data` should be an AttrDict, consider switching to attributized access here:

`name=a_provider.data.provisioning.stacks[0]`

Reply:
Done